packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -456,6 +454,12 @@ export class GarbageCollector implements IGarbageCollector {
             this.mc.config.getBoolean(runSweepKey) ?? (this.sessionExpiryTimeoutMs !== undefined && this.sweepEnabled)
         );
 
+        // Override inactive timeout if test config or gc options to override it is set.
+        this.inactiveTimeoutMs =
+            this.mc.config.getNumber("Fluid.GarbageCollection.TestOverride.InactiveTimeoutMs") ??
+            this.gcOptions.inactiveTimeoutMs ??
```

Comment:
What override is more important? The config or the container options?

Reply:
The config takes preference because they are used for experimentation / testing. The idea is that you can use these to override the actual configs, test things and once they look good, start using configs.